flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/table/OverWindowTest.scala

```diff
@@ -53,20 +53,21 @@ class OverWindowTest extends TableTestBase {
           unaryNode(
             "DataStreamCalc",
             streamTableNode(0),
-            term("select", "a", "b", "c", "proctime")
+            // RexSimplify does not simplify the nested visitCall, due to [CALCITE-2726]
```

Comment:
Can you also link to the open issue? This ensures that we can check for this issue in the future.

Reply:
haven't try opening an incident yet. will try to do that an update the PR